### DISCUSSION TITLE
Bug: can't edit project fields on issues in an archived repository

### DISCUSSION THREAD

**Discussion by simonw** [community/community#19988] [Projects and Issues]
I recently archived one of my GitHub repositories. I now find that I cannot edit the GitHub Projects statuses of project items that are linked to issues in that repository.

Here's an example: I remove the GitHub Projects label, causing the item to drop out of that area... but then a moment later it comes back again because my change was not persisted:

![bug](https://user-images.githubusercontent.com/9599/177194651-eb4e4d6c-5206-40e4-97f2-be6e2beefb2f.gif)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
